notebooks/official/tabular_workflows/tabnet_on_vertex_pipelines.ipynb

```diff
@@ -0,0 +1,1315 @@
+{
```

Comment:
<p>We can define these three parameters here and remove the definitions at the beginning of the "Define training specification" section</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/726/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='44'/>